### PR TITLE
Feature/s4 listing update

### PR DESCRIPTION
**Pull request by LHuss**
Added images to the card display when viewing items.
Refactored Listing and ItemDescription objects as well as their unit tests for this change.

Should complete issue #54

### CODE REVIEW

File: app/src/main/res/layout/item_list_row.xml [hunk 1]
```diff
@@ -1,28 +1,50 @@
...
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:text="name"/>
-        <TextView
-            android:id="@+id/list_item_price"
-            android:layout_width="wrap_content"
+        android:layout_margin="10dp"
+        android:orientation="horizontal">
+
+        <ImageView
```
Comment (kkhh001) [inline app/src/main/res/layout/item_list_row.xml:18]: I like the the proportions you chose for the layout of the item card (image with the text) !

**Review by kkhh001 — APPROVED** [review]
Clean layout and good job on the unit test part 👍

File: app/src/main/java/Fragments/HomeFragment.java [hunk 2]
```diff
@@ -95,10 +95,14 @@ public void onDataChange(DataSnapshot dataSnapshot) {
...
-					    item.setName((String) itemObj.get("Name"));
-					    item.setPrice(((Number)itemObj.get("Price")).doubleValue());
+
+						String id = key;
+						String name = (String) itemObj.get("Name");
+						Double price = ((Number)itemObj.get("Price")).doubleValue();
+						String url = (String) itemObj.get("ImageURL");
+
+					    Listing item = new Listing(key, name, price, url);
+
```
Comment (davidberard2) [inline app/src/main/java/Fragments/HomeFragment.java:105]: Like the addition of the method to create listing. Good job

**Review by davidberard2 — APPROVED** [review]

**Comment by EvanMateo7** [comment]
We'll need to find a way of reducing image files to increase performance..

**Pull request merged**